tests/databases/redis/test_sentinel.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+import redis as redisdb
+
+from testsuite.databases.redis import service
+
+
+@pytest.fixture(scope='session')
+def _redis_service_settings(pytestconfig):
+    return service.get_sentinel_service_settings()
+
+
+def test_sentinel_basic(redis_service, _redis_service_settings: service.ServiceSettings):
+    redis_db = redisdb.StrictRedis(
```

Comment:
issue sentinel specific command here

Reply:
`SENTINEL MASTERS` is a sentinel specific command